scrapi/harvesters/springer.py

```diff
@@ -0,0 +1,153 @@
+"""
+BioMed Central harvester of public projects for the SHARE Notification Service
+Note: At the moment, this harvester only harvests basic data on each article, and does
+not make a seperate request for additional metadata for each record.
+
+Example API query: http://api.springer.com/meta/v1/json?api_key=yourapikey&q=*&s=1&p=200
+"""
+
+from __future__ import unicode_literals
+
+import json
+import logging
+from furl import furl
+from datetime import date, timedelta
+
+from nameparser import HumanName
+
+from scrapi import requests
+from scrapi import settings
+from scrapi.base import JSONHarvester
+from scrapi.linter.document import RawDocument
+from scrapi.base.helpers import build_properties, date_formatter
+
+logger = logging.getLogger(__name__)
+
+
+def process_contributors(authors):
```

Comment:
This has a lot of redundancy with ```default_name_parser```

Might be better to do 
```python
default_name_parser([person['creator'] for person in authors)
```
can do that in line with

```python
compose(default_name_parser, lambda authors: [author['creator'] for author in authors]
```

negating the need for an extra function here entirely.